ABasePawn/ABasePawn.cpp

```diff
@@ -276,10 +276,12 @@ void       ABasePawn::PreInit() {
 
     // set tracking origin to floor
     auto HMD = GEngine->XRSystem.Get();
-    if (HMD)
+    if (HMD) {
         HMD->SetTrackingOrigin(EHMDTrackingOrigin::Floor);
-    else
-        NLogger::Warning("Failed to initialize HMD!");
+    } else {
+        // TODO switch to desktop pawn
+        ADebug::Log("Failed to initialize HMD!");

```

Comment:
This does not necessarily need to be a warning since we intend to switch the main pawn to a desktop pawn if no HMD is found